common/src/main/java/com/wynntils/utils/StringUtils.java

```diff
@@ -37,4 +39,30 @@ public static String uncapitalizeFirst(String input) {
         if (input.length() == 0) return "";
         return Character.toLowerCase(input.charAt(0)) + input.substring(1);
     }
+
+    public static String[] wrapTextBySize(String s, int maxPixels) {
```

Comment:
This function seems misplaced. It should be in like `com.wynntils.mc.utils.StringUtils.java` instead. (If we don't have such a class it needs to be created, or you can shoehorn it into `McUtils` for the time being.) 

The font width is a Minecraft thingy, not a Wynncraft thingy.